src/tests/stats/test_distribution_uniform.fypp

```diff
@@ -0,0 +1,282 @@
+#:include "common.fypp"
+#:set RC_KINDS_TYPES = REAL_KINDS_TYPES + CMPLX_KINDS_TYPES
+#:set ALL_KINDS_TYPES = INT_KINDS_TYPES + RC_KINDS_TYPES
+program test_distribution_uniform
+    use stdlib_error, only : check
+    use stdlib_kinds
+    use stdlib_stats_distribution_PRNG, only : random_seed, dist_rand
+    use stdlib_stats_distribution_uniform,                                      &
+        uni_rvs => uniform_distribution_rvs,                                    &
+        uni_pdf => uniform_distribution_pdf,                                    &
+        uni_cdf => uniform_distribution_cdf
```

Comment:
Since you rename already the functions, why not use the shorter names for these functions (e.g., uni_rvs instead of uniform_distribution_rvs (or uniform_rvs))?

Reply:
Longer names are more meaningful to users as lib procedures. Users can always shorten the name when use it, such as uni => uniform_distribution_rvs